book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -526,76 +526,76 @@ index 0399cd5,59727f0..e1d0799
 
-===== Fix the references
+===== Ayusin ang mga reperensiya
 
-If the unwanted merge commit only exists on your local repository, the easiest and best solution is to move the branches so that they point where you want them to.
-In most cases, if you follow the errant `git merge` with `git reset --hard HEAD~`, this will reset the branch pointers so they look like this:
+Kung ang hindi gustong merge commit ay umiiral na lamang sa iyong lokal na repositoryo, ang pinakamadali at pinakamahusay na solusyon ay ang ilipat ang mga branch upang sila ay nagtuturo kung saan gusto mo sila.
+Sa maraming mga kaso, kung ikaw ay sumusunod sa maling `git merge` na may `git reset --hard HEAD~`, ito ay nagre-reset sa mga punto ng branch para magmukha silang ganito:
 
-.History after `git reset --hard HEAD~`
-image::images/undomerge-reset.png[History after `git reset --hard HEAD~`.]
+.Kasaysayan pagkatapos ang `git reset --hard HEAD~`
+image::images/undomerge-reset.png[Kasaysayan pagkatapos ang `git reset --hard HEAD~`.]
 
-We covered `reset` back in <<_git_tools#_git_reset>>, so it shouldn't be too hard to figure out what's going on here.
-Here's a quick refresher: `reset --hard` usually goes through three steps:
+Tinatalakay namin ang `reset` na balik sa <<_git_tools#_git_reset>>, kaya ito ay hindi napakahirap na malaman kung ano nangyayari dito.
+Narito ang isang mabilis na refresher: `reset --hard` kadalasang napupunta sa tatlong paraan:
 
-. Move the branch HEAD points to.
-  In this case, we want to move `master` to where it was before the merge commit (`C6`).
-. Make the index look like HEAD.
-. Make the working directory look like the index.
+. Igalaw ang branch HEAD na mga puntos sa.
+  sa kasong ito, gusto naming ilipat ang `master` sa kung saan ito ay nating na merge commit (`C6`).
+. Gawin ang index na kamukha ng HEAD.
+. Gawin ang tinatrabahong direktoryo na kamukha ng index.
 
-The downside of this approach is that it's rewriting history, which can be problematic with a shared repository.
-Check out <<_git_branching#_rebase_peril>> for more on what can happen; the short version is that if other people have the commits you're rewriting, you should probably avoid `reset`.
-This approach also won't work if any other commits have been created since the merge; moving the refs would effectively lose those changes.
+Ang downside sa diskarteng ito ay ang kanyang sinusulatang muli na kasaysayan, na maaaring maging problema na may isang ibinabahaging direktoryo.
+Tingnan ang <<_git_branching#_rebase_peril>> para sa marami na mga pwedeng mangyari; ang maikling bersyon ay kung ang ibang tao ay mayroong mga commit na iyong sinusulatang muli, dapat mong marahil iwasan ang `reset`.
+Ang pamamaraan na ito ay hindi rin gagana kung ang ibang mga anumang commit ay nalikha mula nang na-merge; paglilipat sa mga ref ay epektibong mawawala ang mga pagbabago na iyon.
 
 [[_reverse_commit]]
-===== Reverse the commit
+===== Baliktad na commit
```

Comment:
`Baliktad na commit` to `Ibaliktad ang commit`

Reply:
done change